Jenkinsfile

```diff
@@ -125,6 +126,16 @@ pipeline {
             sh 'nix-shell --run "cargo test"'
           }
         }
+        stage('chart publish test') {
+          when {
+            expression { helm_test == true }
+          }
+          agent { label 'nixos-mayastor' }
+          steps {
+            sh 'printenv'
+            sh 'nix-shell --run "./scripts/helm/test-publish-chart-yaml.sh"'
```

Comment:
please run it with `--pure` command to ensure sem-ver & jq are picked up only from nix environ(I guess this will be default in most cases as `$PATH` usually has the nix-path in the front

Reply:
Yeah it should move to the front of the PATH. Using pure is most useful ensure we're not using any tool/lib which is not defined in the nix, otherwise it might not work on someone else's box.
I'll add pure here and in the actions :+1: